src/MICore/Debugger.cs

```diff
@@ -482,10 +497,17 @@ public Task CmdStart()
             return CmdAsync("-exec-run", ResultClass.running);
         }
 
-        protected bool _requestingRealAsyncBreak = false;
-        public Task CmdBreak()
+        public enum BreakRequest
+        {
+            None,
+            Async,
+            Stop,
+            Internal
+        }
+        protected BreakRequest _requestingRealAsyncBreak = BreakRequest.None;
+        public Task CmdBreak(BreakRequest request)
         {
-            _requestingRealAsyncBreak = true;
+            _requestingRealAsyncBreak = request;
```

Comment:
Yes, I must have screwed up the add for this file.  Will fix.


_____________________________
From: Gregg Miskelly <notifications@github.com<mailto:notifications@github.com>>
Sent: Monday, May 23, 2016 7:51 PM
Subject: Re: [Microsoft/MIEngine] Child2 (#329)
To: Microsoft/MIEngine <miengine@noreply.github.com<mailto:miengine@noreply.github.com>>
Cc: Paul Maybee <paulmay@microsoft.com<mailto:paulmay@microsoft.com>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>



In src/MICore/Debugger.cs<https://github.com/Microsoft/MIEngine/pull/329#discussion_r64321625>:

>          {> -            _requestingRealAsyncBreak = true;> +            _requestingRealAsyncBreak = request;

Did you forget to add your changes? This is still just overwriting the variable.

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/MIEngine/pull/329/files/1e0e5486fa6440bcea7c7bc67e0da514d5f2641d#r64321625>